usage/jsgui/src/main/license/source-inclusions.yaml

```diff
@@ -38,3 +38,5 @@
 - id: js-uri
 - id: js-yaml.js
 - id: jquery.form.js
+
+- id: codemirror.js
```

Comment:
no need to add the codemirror dependencies here, but we do need a corresponding entry in `usage/dist/licensing/overrides.yaml` and there under `files` list the files pulled in (as done for `jquery.dataTables`) -- it's only used to generate the proper license files